data/spells/scripts/support/find_person.lua

```diff
@@ -46,6 +46,7 @@ function onCastSpell(creature, variant)
 	end
 
 	local playerPosition = creature:getPosition()
+	local targetPosition = targetPlayer:getPosition()
```

Comment:
But I can still not get it to throw an error, so how is it bugged?
We still have the text variable.

I tested casting it without any name, gives me the cancel message, same a misspelled name.
As I said, the script works, so please tell me how to make it bug.

And I highly doubt it would even throw an error, since if the variant dosn't have a string variable it would return false since it can't get the userdata for targetPlayer.